GameData/ROCapsules/PartConfigs/Apollo/ApolloLES.cfg

```diff
@@ -7,9 +7,9 @@ PART
 	RSSROConfig = true
 	RP0conf = true
```

Comment:
Don't set RP0 configuration state.
The only change to this file was whitespace.